crypto/pemutil/pem_test.go

```diff
@@ -259,7 +259,7 @@ func TestReadCertificateBundle(t *testing.T) {
 		{"testdata/notexists.crt", 0, errors.New("open testdata/notexists.crt failed: no such file or directory")},
 		{"testdata/badca.crt", 0, errors.New("error parsing testdata/badca.crt")},
 		{"testdata/badpem.crt", 0, errors.New("error decoding PEM: file 'testdata/badpem.crt' contains unexpected data")},
-		{"testdata/badder.crt", 0, errors.New("error parsing testdata/badder.crt: asn1: syntax error: data truncated")},
+		{"testdata/badder.crt", 0, errors.New("error parsing testdata/badder.crt::")},
```

Comment:
There's an extra `:`
```suggestion
		{"testdata/badder.crt", 0, errors.New("error parsing testdata/badder.crt:")},
```